Dockerfile.deps

```diff
@@ -1,6 +1,6 @@
 # Build libwally wasm files. Used for client-side blinding verification on Elements-based chains
 
-FROM greenaddress/wallycore@sha256:d63d222be12f6b2535e1548aa7f21cf649e2230d6c9e01bd518c23eb0bccd46f AS libwally-wasm
+FROM blockstream/wallycore@sha256:6a41958eefaa3920846e423885e807d41eb0f1ae54e28f039d62d5c0f97003f5 AS libwally-wasm
```

Comment:
Should this now become 956b107d688f549c6e3884424991b7d3d34d84173990d43046fd760d7918db7c after https://github.com/ElementsProject/libwally-core/pull/467 ?

Reply:
Yes, super new image. Still checking with Jon - looks like there now is _only_ an ARM image.

Reply:
One more followup: https://github.com/ElementsProject/libwally-core/pull/468
and noticed that they pushed either AMD or ARM, but we can't have an if/else here so still using a blockstream source for now.